knowledge_runtime/start.sh

```
@@ -0,0 +1,295 @@
+echo -e "${GREEN}✓ Python $PYTHON_VERSION detected${NC}"
+echo ""
+
+# Step 2: Check uv installation
+echo -e "${BLUE}[2/4] Checking uv installation...${NC}"
+if ! command -v uv &> /dev/null; then
+    echo -e "${YELLOW}Warning: uv is not installed${NC}"
+    echo -e "${YELLOW}Installing uv...${NC}"
+    curl -LsSf https://astral.sh/uv/install.sh | sh
+
+    if [ -f "$HOME/.cargo/env" ]; then
+        source "$HOME/.cargo/env"
```

Shellcheck (0.11.0)

[info] Not following: ./.cargo/env was not specified as input (see shellcheck -x).

(SC1091)